qiita_pet/handlers/study_handlers/prep_template.py

```diff
@@ -0,0 +1,79 @@
+# -----------------------------------------------------------------------------
```

Comment:
One of the ideas of these changes was to create tests, right? However, there are no tests for this ...